### PR TITLE
vmaf: Add package

### PR DESCRIPTION
**Pull request by Biswa96**
Please review.

### CODE REVIEW

**Comment by Biswa96** [comment]
Should the dll name be libvmaf.dll or libvmaf-1.dll? @1480c1

File: mingw-w64-vmaf/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=2.3.0
+pkgrel=1
+pkgdesc='Perceptual video quality assessment algorithm based on multi-method fusion (mingw-w64)'
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
+url='https://github.com/Netflix/vmaf/'
+license=('BSD')
+depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs"
```
[the remark this answers is not in the document]
Comment (Biswa96) [reply mingw-w64-vmaf/PKGBUILD:13]: The clang ones does not link with openmp, just libc++ and libunwind.

[the remark this answers is not in the document]
Comment (kmilos) [reply mingw-w64-vmaf/PKGBUILD:13]: Maybe they would if you installed openmp, like in the trial case of mlpack (and gmic)...

[the remark this answers is not in the document]
Comment (kmilos) [reply mingw-w64-vmaf/PKGBUILD:13]: Do the gcc builds link against libgomp? If not, then never mind of course. I've seen one of the vmaf 3rd party libraries use OpenMP, so I assumed they do...

[the remark this answers is not in the document]
Comment (Biswa96) [reply mingw-w64-vmaf/PKGBUILD:13]: I have compiled this in my PC and checked the dependencies with openmp installed. Also the build process does not search for openmp. Whereas gmic (for example) searches for openmp:

```
-- Found OpenMP_C: -fopenmp (found version "4.5")
-- Found OpenMP_CXX: -fopenmp (found version "4.5")
-- Found OpenMP: TRUE (found version "4.5")
```

[the remark this answers is not in the document]
Comment (Biswa96) [reply mingw-w64-vmaf/PKGBUILD:13]: No problem. Thanks for the review.

**Comment by 1480c1** [comment]
> Should the dll name be libvmaf.dll or libvmaf-1.dll? @1480c1

I would say leave it as the default, no preference either way

**Pull request merged**